apps/yarn.lock

```diff
@@ -18683,6 +18684,11 @@ yargs@~3.10.0:
     decamelize "^1.0.0"
     window-size "0.1.0"
 
+yarn@^1.22.10:
```

Comment:
I am curious why this was added after running `yarn` from the `apps` directory?

Reply:
Because the version of `ml-playground` has yarn as a dependency and not a devDependency. https://github.com/code-dot-org/ml-playground/blob/main/package.json#L83

Reply:
Ideally, `ml-playground` would not have yarn specifically as a dependency... so that it won't become strictly a dependency here.